docker/optimism-scripts/utils.py

```diff
@@ -22,20 +21,24 @@
 OPTIMISM_L2_MESSENGER_ADDRESS = "0x4200000000000000000000000000000000000007"
 
 
+def _address_file_path() -> str:
+    return os.environ["ADDRESS_FILE_PATH"]
```

Comment:
Can you add a default here?

Reply:
I would actually prefer the script to fail here if the env variable is not set, so that we know about it and can fix whatever is invoking the script.